viam_cartographer.go

```diff
@@ -187,32 +187,31 @@ func New(
 		return nil, err
 	}
 
+	// Get the lidar for the Dim2D cartographer sub algorithm
 	lidarName := svcConfig.Camera["name"]
```

Comment:
I'm realizing I'm used to the styling in rdk  - if the config has a Camera, I assume it to be something like:
```go
Camera {
     Name string. `json:"name"`
     DataFrequency int `json:"data_frequency_hz,omitempty"`
}
```

So it's cleaner to validate and access and you don't have to remember the key in the string all over the place

but now I see in `config.go` the attributes are maps rather than struts that can be serialized. I don't like it but it's a minor cleanup point. Don't do in this pr.

Okay to leave as is.

Reply:
Should we create a ticket for this?

Reply:
No, it's different, but I don't really care that it's different. Even if I like the in rdk styling more.